src/main/flight/servos.c

```diff
@@ -296,20 +296,54 @@ STATIC_UNIT_TESTED void forwardAuxChannelsToServos(uint8_t firstServoIndex)
     }
 }
 
+static void biquadFilterInitd1d2(biquadFilter_t *filter, float sample)
+{
+    filter->d2 = filter->b2 * sample - filter->a2 * sample;
+    filter->d1 = filter->b1 * sample - filter->a1 * sample + filter->d2;
+}
+
 static void filterServos(void)
 {
-    if (servoConfig()->servo_lowpass_freq) {
+    static uint16_t rudderFilterFreq;
+
+    if (servoConfig()->servo_lowpass_freq || servoConfig()->tri_servo_lpf_hz) {
```

Comment:
What happens if `servo_lowpass_freq == 0` and `tri_servo_lpf_hz != 0`? 
From the code I see that wervo LPF will be initialized with zero cutoff frequency. I presume this is not the intention.